cosmos/stride.json

```diff
@@ -99,63 +113,152 @@
+        "coinDecimals": 6,
+        "coinGeckoId": "cosmos",
+        "gasPriceStep": {
+          "low": 0.0001,
+          "average": 0.001,
+          "high": 0.01
+        }
+      },
+      {
+        "coinDenom": "OSMO",
+        "coinMinimalDenom": "ibc/D24B4564BCD51D3D02D9987D92571EAC5915676A9BD6D9B0C1D0254CB8A5EA34",
+        "coinDecimals": 6,
+        "coinGeckoId": "osmosis",
+        "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/osmosis/uosmo.png",
+        "gasPriceStep": {
+          "low": 0.001,
+          "average": 0.01,
+          "high": 0.1
+        }
+      },
+      {
+        "coinDenom": "stOSMO",
+        "coinMinimalDenom": "stuosmo",
+        "coinDecimals": 6,
+        "coinGeckoId": "osmosis",
+        "gasPriceStep": {
+          "low": 0.001,
+          "average": 0.01,
+          "high": 0.1
+        }
+      },
+      {
+        "coinDenom": "stATOM",
+        "coinMinimalDenom": "stuatom",
+        "coinDecimals": 6,
+        "coinGeckoId": "stride-staked-atom",
+        "gasPriceStep": {
+          "low": 0.0001,
+          "average": 0.001,
+          "high": 0.01
+        }
+      },
+      {
+        "coinDenom": "INJ",
+        "coinMinimalDenom": "ibc/A7454562FF29FE068F42F9DE4805ABEF54F599D1720B345D6518D9B5C64EA6D2",
+        "coinDecimals": 18,
+        "coinGeckoId": "injective-protocol",
+        "gasPriceStep": {
+          "low": 500000000,
+          "average": 500000000,
+          "high": 500000000
+        }
+      },
+      {
+        "coinDenom": "stINJ",
+        "coinMinimalDenom": "stinj",
+        "coinDecimals": 18,
+        "coinGeckoId": "stride-staked-injective",
+        "gasPriceStep": {
+          "low": 500000000,
+          "average": 500000000,
+          "high": 500000000
+        }
+      },
+      {
+        "coinDenom": "EVMOS",
+        "coinMinimalDenom": "ibc/4B322204B4F59D770680FE4D7A565DDC3F37BFF035474B717476C66A4F83DD72",
+        "coinDecimals": 18,
+        "coinGeckoId": "evmos",
+        "gasPriceStep": {
+          "low": 20000000000,
+          "average": 20000000000,
+          "high": 20000000000
+        }
+      },
+      {
+        "coinDenom": "stEVMOS",
+        "coinMinimalDenom": "staevmos",
+        "coinDecimals": 18,
+        "coinGeckoId": "stride-staked-evmos",
+        "gasPriceStep": {
+          "low": 20000000000,
+          "average": 20000000000,
+          "high": 20000000000
+        }
+      },
+      {
+        "coinDenom": "TIA",
+        "coinMinimalDenom": "ibc/BF3B4F53F3694B66E13C23107C84B6485BD2B96296BB7EC680EA77BBA75B4801",
+        "coinDecimals": 6,
+        "coinGeckoId": "celestia",
+        "gasPriceStep": {
+          "low": 0.01,
+          "average": 0.01,
+          "high": 0.01
+        }
+      },
+      {
+        "coinDenom": "stTIA",
+        "coinMinimalDenom": "stutia",
+        "coinDecimals": 6,
+        "coinGeckoId": "",
```

Comment:
@riley-stride Could you remove empty `coinGeckoId` field?

Reply:
done